lib/experimental/media/load.php

```diff
@@ -236,16 +254,11 @@ function gutenberg_start_cross_origin_isolation_output_buffer(): void {
 	$coep = $is_safari ? 'require-corp' : 'credentialless';
 
 	ob_start(
-		function ( string $output, ?int $phase ) use ( $coep ): string {
-			// Only send the header when the buffer is not being cleaned.
-			if ( ( $phase & PHP_OUTPUT_HANDLER_CLEAN ) === 0 ) {
```

Comment:
Can you explain why this is no longer relevant?

I wouldn't have asked if it hadn't been here before, but why was it there before and is now being removed?

Reply:
Well it was there before because I thought I needed it, but turns out it doesn't work, so I just removed it.

Reply:
FWIW, the PL plugin includes this: https://github.com/WordPress/performance/pull/1443

And the Optimization Detective plugin: https://github.com/WordPress/performance/pull/1317

In my testing it worked, although I wrestled with this one for awhile to replicate the scenario where it would be relevant.

Reply:
IDK what's different here, but this change here fixes the issue for me and would unblock further work, so I'd appreciate an approval